sdks/python/apache_beam/transforms/trigger_test.py

```diff
@@ -81,16 +84,8 @@ def bundle_data(data, size):
 
     if not groupings:
       groupings = [1]
+    grouping = list(groupings) + [-group_by for group_by in groupings]
```

Comment:
<!--thread_id:cc_175236867_t; commit:23bbd5d9bf76313060d10b6a040be8c452229d64; resolved:1-->
<!--section:context-quote-->
> **robertwb** wrote:
> Oh, yes, nice eye!
> 
> And fixing this makes some of the non-deterministic ones fail... Enabling only explicitly on some tests.

<!--section:body-->
Thanks!  Could you also add a comment at the beginning of this method on the behavior of `groupings`?  It is not obvious without a close reading.